pages/project/update-licence-number/index.js

```diff
@@ -0,0 +1,35 @@
+const isUUID = require('uuid-validate');
+const { page } = require('@asl/service/ui');
+const confirm = require('./routers/confirm');
+const update = require('./routers/update');
+const { NotFoundError } = require('../../../lib/errors');
+
+module.exports = settings => {
+  const app = page({
+    ...settings,
+    root: __dirname,
+    paths: ['/confirm']
+  });
+
+  app.use((req, res, next) => {
+    if (!isUUID(req.projectId)) {
+      return next(new NotFoundError());
+    }
+
+    return req.api(`/establishment/${req.establishmentId}/projects/${req.projectId}`)
+      .then(({ json: { data, meta } }) => {
+        req.project = data;
```

Comment:
Would be good to check that it's a stub at this point and 404 if not.

Reply:
done.